plugins/wporg-5ftf/includes/pledge-form.php

```diff
@@ -232,7 +232,26 @@ function process_form_manage( $pledge_id, $auth_token ) {
 
 	PledgeMeta\save_pledge_meta( $pledge_id, $submission );
 
-	// @todo Upload & attach logo.
+	if ( isset( $_FILES['org-logo'], $_FILES['org-logo']['tmp_name'] ) && ! empty( $_FILES['org-logo']['tmp_name'] ) ) {
+		$current_logo_id    = get_post_thumbnail_id( $pledge_id );
+		$logo_attachment_id = upload_image( $_FILES['org-logo'] );
+		if ( is_wp_error( $logo_attachment_id ) ) {
+			return $logo_attachment_id;
+		}
+
+		// Attach new logo to the pledge.
+		wp_update_post( array(
+			'ID'          => $logo_attachment_id,
+			'post_parent' => $pledge_id,
+		) );
+		$updated = set_post_thumbnail( $pledge_id, $logo_attachment_id );
+
+		// Trash the old logo.
+		if ( $updated ) {
+			wp_delete_attachment( $current_logo_id );
```

Comment:
Slightly pedantic, but I think this would read better if the variable was `$original_logo_id` instead of `current`. That way when you look at this line you know you're deleting the old logo, not the new, "current" one.